host_os.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) IBM Corp. 2016.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import logging
+import os
+import sys
+
+from lib import utils
+from tools import build_iso
+from tools import build_package
+from tools import create_release_notes
+from tools import setup_environment
+from tools import upgrade_versions
```

Comment:
I'm fine with `tools`, but just to leave it registered, I think another good option would be `lib/commands`.